helpers/TXTHelper.py

```diff
@@ -0,0 +1,15 @@
+import copy
+from datetime import datetime
+import hashlib
+import json
+import uuid
+
+
+def generate_TXT_File_plain(deduplicated_observations):
+    dt = datetime.now()
```

Comment:
dt defined but not used, i guess intent was timestamp for file, but this now done in TIELoader.py